data/json/npcs/talk_tags.json

```
@@ -901,7 +1057,9 @@
     "text": [
       "I need some batteries to power my CBMs.",
       "I can't recharge my CBMs without some batteries.",
-      "Hey, <name_g>, can I get some batteries here?  I need to recharge."
+      "Hey, <name_g>, can I get some batteries here?  I need to recharge.",
+      "Internal batteries running low. Recharge needed.",
+      "Power going low, my internal system will malfunction without a recharge"
```

```suggestion
      "Power going low, my internal system will malfunction without a recharge."
```
Not necessarily true. You should remove this one.